packages/mongo-menu/src/MongoMenu.tsx

```diff
@@ -206,6 +211,7 @@ interface MongoMenuProps {
  */
 function MongoMenu({
   user: { name, email },
+  accountURL = 'https://cloud.mongodb.com/account/profile',
```

Comment:
This URL doesn't yet exist, and it also might not be the final location.

The current location is `https://cloud.mongodb.com/v2#/account`, so perhaps we default to that until we have the actual working new URL?

Reply:
👍

Reply:
Ahh, @Faeylayn was saying that this was where it was gonna live until it was moved to `account.mongodb.com`. Just wanna double-check this.

Reply:
We can agree that defaulting the url at currently to v2#/account makes sense, currently the account/profile is only going to exist in a placeholder setup for a while now. This just means we'll need to have another release to change this to the URL we will be using while the account App is a part of MMS and then another change for when the account app is separated from MMS. Currently the App is being built where it will exist at https://cloud.mongodb.com/account/profile and then will move to the account.mongodb.com at some point in the far flung future

Reply:
👍